components/app/app.js

```diff
@@ -23,7 +23,7 @@ import TagManager from 'react-gtm-module'
 import {
   AuthorizationProvider,
   FirebaseProvider,
-  Icons,
+  // Icons,
```

Comment:
@majid-amiri Please remove the commented out code

Reply:
Good point.